test/includes.specs.ts

```diff
@@ -0,0 +1,23 @@
+import * as chai from 'chai'
+import 'mocha'
+chai.should()
+import fc from 'fast-check'
+import { includes } from '../src/includes'
+import { arrayAndNotElement } from './generators'
+
+function generateRandom(maxLimit: number): number{
```

Comment:
not wrong in itself but take too much lines ^^
```ts
const generateRandom = (maxLimit: number): number =>
  Math.floor(Math.random() * maxLimit)
```

Reply:
Ok